build.gradle

```
@@ -124,6 +124,22 @@ class AddressBookTest extends Test {
     public AddressBookTest() {
     	  forkEvery = 1
         systemProperty 'testfx.setup.timeout', '60000'
+
+        /**
+         * The following code prints the current running test's name in the
+         * CI's log.
+         *
+         * The default behaviour of Gradle does not print out the name of
+         * the test being executed. This makes it hard to determine if
+         * the tests are running, or are actually being skipped and happens
+         * to be reported as fine. It is also hard to figure out which test
+         * borked the CI, which may happen occasionally.
```

I think all these can be easily shortened to `Print the currently running test name in the CI build log so we can check if tests are being silently skipped or stalling the build.`

Let's try to keep comments as succinct as possible, otherwise the code would become to noisy and hard to read. Readers who want a more detailed explanation can simply `git-blame` this chunk of code.